src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/PatchCorrector.kt

```diff
@@ -49,4 +84,5 @@ object PatchCorrector {
     }
 
     public class DeletePatchException(message: String) : Throwable(message)
+    public class ApplyPatchException(message: String) : Throwable(message)
```

Comment:
You throw this exception but never catch it, so from the usage perspective it is no different than just RuntimeException or NullPointerException or whatever. The goal of introducing your own class must be ability to catch and handle exceptions of that class.